src/core/api.js

```diff
@@ -5,10 +5,16 @@ const ApiService = {
     axios.defaults.baseURL = "http://react-cdp-api.herokuapp.com/";
   },
   getMovies(params) {
-    return axios.get("movies", { params: params }).then(result => result.data);
+    return axios.get("movies", { params: params }).then(response => {
+      console.log(response);
+      return response.data;
+    });
   },
   getMoviesById(id) {
-    return axios.get("movies" + `/${id}`);
+    return axios.get("movies" + `/${id}`).then(response => {
```

Comment:
"movies" is considered an endpoint and could be a subject to change in the app lifecycle.
Consider extracting this endpoint details and putting it as MOVIES_ENDPOINT for example under config directory.